atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -69,15 +69,18 @@ def get_koji_upload_dir(workflow):
     return workspace[WORKSPACE_KEY_UPLOAD_DIR]
 
 
-def override_build_kwarg(workflow, k, v):
+def override_build_kwarg(workflow, k, v, platform=None):
     """
     Override a build-kwarg for all worker builds
     """
     key = OrchestrateBuildPlugin.key
+    if not platform:
+        platform = 'None'
```

Comment:
It should be `None` not `'None'`.

Reply:
Indexing things by None makes my head hurt, but it works.

Fixed.